itest/lib/appStep/api/savePcap.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import {Application} from "spectron"
+
+import {retryUntil} from "../../control"
+import {waitForClickableButtonAndClick} from "./click"
+import logStep from "../util/logStep"
+import {selectors} from "../../../../src/js/test/integration"
+
+export default async (app: Application) => {
+  await waitForClickableButtonAndClick(app, selectors.pcaps.button)
+  return await logStep("wait for a download to finish", async () => {
+    await app.client.waitForVisible(selectors.downloadMessage)
+    return await retryUntil(
+      () => app.client.getText(selectors.downloadMessage),
+      (text) => text == "Download Complete" || text.includes("Download error")
+    )
+  })
+}
+
+export const pcapsDir = async (app: Application) =>
+  await app.electron.remote.app.getPath("temp")
```

Comment:
Building off the comment above, I guess a general rule could be that if you are returning a promise, there is no need to await it right before you return it.  Await should be used only as an alternative to a `.then()` call.

This line above would function identically to:

```js
export const pcapsDir = (app) => app.electron.remote.app.getPath("temp")
```

Reply:
This is a helpful comment. Fixed.